en/flight_controller/cuav_v5_plus.md

```diff
@@ -150,6 +151,11 @@ Some JTAG use this voltage to set the IO levels when communicating to the target
 
 For direct connection to *Segger Jlink* we recommended you use the 3.3 Volts of DSM/SBUS/RSSI pin 4 as Pin 1 on the debug connector (`Vtref`).
 
+#### The HV\_PM power module output is unfused {#issue_pm_unfused}
+
+The *HV\_PM* power module supplied with the kit is not protected by a fuse:
+- Only connect devices with the power turned off.
+- The power module may be damaged if connected incorrectly/too much power is drawn.
 
```

Comment:
It is not the PM - it is all the things including the person 

Add: Improper wiring can lead to equipment damage or personal harm.

Reply:
Yes. Done consistently in https://github.com/PX4/px4_user_guide/pull/538#discussion_r307573962